ecosystem/platform/server/app/views/it2_profiles/_form.html.erb

```diff
@@ -78,7 +78,7 @@
           </div>
 
           <div class="mb-6">
-          <%= f.label :fullnode_network_key, class: 'font-mono uppercase block mb-2' %>
+          <%= f.label :fullnode_network_key, 'Fullnode Network Public Key', class: 'font-mono uppercase block mb-2' %>
```

Comment:
I would either remove "Public" from "Fullnode Network Public Key" or alternatively add "Public" to all of the other fields